nft_ingester/src/program_transformers/bubblegum/mint_v1.rs

```diff
@@ -232,43 +232,79 @@ where
+                    let mut db_creator_infos = Vec::with_capacity(creators.len());
+
+                    // Vec to hold info on whether a creator is verified.  This info is protected by `seq` number.
+                    let mut db_creator_verified_infos = Vec::with_capacity(creators.len());
+
+                    // Set to prevent duplicates.
                     let mut creators_set = HashSet::new();
+
                     for (i, c) in creators.iter().enumerate() {
                         if creators_set.contains(&c.address) {
                             continue;
                         }
-                        db_creators.push(asset_creators::ActiveModel {
+                        db_creator_infos.push(asset_creators::ActiveModel {
                             asset_id: Set(id_bytes.to_vec()),
                             creator: Set(c.address.to_bytes().to_vec()),
+                            position: Set(i as i16),
                             share: Set(c.share as i32),
+                            slot_updated: Set(Some(slot_i)),
+                            ..Default::default()
+                        });
+
+                        db_creator_verified_infos.push(asset_creators::ActiveModel {
+                            asset_id: Set(id_bytes.to_vec()),
+                            creator: Set(c.address.to_bytes().to_vec()),
                             verified: Set(c.verified),
-                            seq: Set(seq as i64), // do we need this here @micheal-danenberg?
-                            slot_updated: Set(slot_i),
-                            position: Set(i as i16),
+                            seq: Set(Some(seq as i64)),
                             ..Default::default()
                         });
+
                         creators_set.insert(c.address);
                     }
 
-                    let query = asset_creators::Entity::insert_many(db_creators)
+                    // This statement will update base information for each creator.
+                    let query = asset_creators::Entity::insert_many(db_creator_infos)
                         .on_conflict(
                             OnConflict::columns([
                                 asset_creators::Column::AssetId,
-                                asset_creators::Column::Position,
+                                asset_creators::Column::Creator,
                             ])
                             .update_columns([
-                                asset_creators::Column::Creator,
+                                asset_creators::Column::Position,
                                 asset_creators::Column::Share,
+                                asset_creators::Column::SlotUpdated,
+                            ])
+                            .to_owned(),
+                        )
+                        .build(DbBackend::Postgres);
+                    txn.execute(query).await?;
```

Comment:
does this query need a where check for example on SlotUpdated?

Reply:
if it cannot be reordered, then is the update needed atg all?

Reply:
It is still needed to be an update because if 'verify_creator` is processed first, then there will be a record but it won't actually have all this base information from the time of minting.

Reply:
> does this query need a where check for example on SlotUpdated?

It does not need protection because the base creator info is ONLY filled in at time of mint.

Reply:
We will likely be updating this to support changes to creator array and collection.